### PR TITLE
🌐 Add Portuguese translation for `docs/deployment/deta.md`

### PR DESCRIPTION
**Pull request by stlucasgarcia**
Hey 👋🏻

This PR adds the Portuguese translation of `deployment/deta.md`.

### CODE REVIEW

File: docs/pt/docs/deployment/deta.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+### Requisitos
+
+Agora, no mesmo diretório crie o arquivo `requirements.txt` com:
+
+```text
+fastapi
+```
+
+!!! tip "Dica"
+    Você não precisa instalar Uvicorn para realizar a implantação na Deta, embora você provavelmente vai querer instalá-lo para testar seu aplicativo localmente.
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:43]:
```suggestion
    Você não precisa instalar Uvicorn para realizar a implantação na Deta, embora provavelmente queira instalá-lo para testar seu aplicativo localmente.
```

File: docs/pt/docs/deployment/deta.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+## Crie uma conta gratuita na Deta
+
+Agora crie <a href="https://www.deta.sh/?ref=fastapi" class="external-link" target="_blank">uma conta gratuita na Deta</a>, você precisará apenas de um email e senha.
+
+Você nem precisa de um cartão de crédito.
+
+## Instale a CLI
+
+Depois de ter sua conta criada, instale Deta <abbr title="Command Line Interface application">CLI</abbr>:
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:63]:
```suggestion
Depois de ter sua conta criada, instale Deta <abbr title="Interface de Linha de Comando">CLI</abbr>:
```

File: docs/pt/docs/deployment/deta.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+Available Commands:
+  auth        Change auth settings for a deta micro
+
+...
+```
+
+</div>
+
+!!! tip "Dica"
+    Se você tiver problemas ao instalar a CLI, verifique a <a href="https://docs.deta.sh/docs/micros/getting_started?ref=fastapi" class="external-link" target="_blank">documentação oficial Deta</a>.
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:110]:
```suggestion
    Se você tiver problemas ao instalar a CLI, verifique a <a href="https://docs.deta.sh/docs/micros/getting_started?ref=fastapi" class="external-link" target="_blank">documentação oficial da Deta</a>.
```

File: docs/pt/docs/deployment/deta.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+
+## Logue com a CLI
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:112]:
```suggestion
## Login pela CLI
```

File: docs/pt/docs/deployment/deta.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+
+Agora faça login na Deta da CLI com:
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:114]:
```suggestion
Agora faça login na Deta pela CLI com:
```

File: docs/pt/docs/deployment/deta.md [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+        "http_auth": "enabled"
+}
+```
+
+!!! tip "Dica"
+    Sua implantação terá um URL `"endpoint"` diferente.
+
+## Confira
+
+Agora, abra seu navegador no URL do `endpoint`. No exemplo acima foi `https://qltnci.deta.dev`, mas o seu será diferente.
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:178]:
```suggestion
Agora, abra seu navegador na URL do `endpoint`. No exemplo acima foi `https://qltnci.deta.dev`, mas o seu será diferente.
```

File: docs/pt/docs/deployment/deta.md [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+}
+```
+
+Agora vá para o `/docs` da sua API, no exemplo acima seria `https://qltnci.deta.dev/docs`.
+
+Ele mostrará sua documentação como:
+
+<img src="/img/deployment/deta/image01.png">
+
+## Permitindo acesso público
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:194]:
```suggestion
## Permitir acesso público
```

File: docs/pt/docs/deployment/deta.md [hunk 8]
```diff
@@ -0,0 +1,258 @@
...
+
+```console
+$ deta auth disable
+
+Successfully disabled http auth
+```
+
+</div>
+
+Agora você pode compartilhar esse URL com qualquer pessoa e eles conseguirão acessar sua API. 🚀
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:210]:
```suggestion
Agora você pode compartilhar essa URL com qualquer pessoa e elas conseguirão acessar sua API. 🚀
```

File: docs/pt/docs/deployment/deta.md [hunk 9]
```diff
@@ -0,0 +1,258 @@
...
+
+## HTTPS
+
+Parabéns! Você realizou a implantação do seu app FastAPI na Deta! 🎉 🍰
+
+Além disso, observe que o Deta lida corretamente com HTTPS para você, para que você não precise cuidar disso e tenha certeza de que seus clientes terão uma conexão criptografada segura. ✅ 🔒
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:216]:
```suggestion
Além disso, observe que a Deta lida corretamente com HTTPS para você, para que você não precise cuidar disso e tenha a certeza de que seus clientes terão uma conexão criptografada segura. ✅ 🔒
```

File: docs/pt/docs/deployment/deta.md [hunk 10]
```diff
@@ -0,0 +1,258 @@
...
+
+## Verifique o Visor
+
+Da UI da sua documentação (você estará em um URL como `https://qltnci.deta.dev/docs`) envie um request para *operação de rota* `/items/{item_id}`.
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:220]:
```suggestion
Na UI da sua documentação (você estará em um URL como `https://qltnci.deta.dev/docs`) envie um request para *operação de rota* `/items/{item_id}`.
```

File: docs/pt/docs/deployment/deta.md [hunk 11]
```diff
@@ -0,0 +1,258 @@
...
+
+Você verá uma aba com "Detalhes", e também a aba "Visor", vá para "Visor".
+
+Lá você pode inspecionar as solicitações recentes enviadas ao seu aplicativo.
+
+Você também pode editá-los e reproduzi-los novamente.
+
+<img src="/img/deployment/deta/image02.png">
+
+## Aprenda mais
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:236]:
```suggestion
## Saiba mais
```

File: docs/pt/docs/deployment/deta.md [hunk 12]
```diff
@@ -0,0 +1,258 @@
...
+
+Em algum momento, você provavelmente desejará armazenar alguns dados para seu aplicativo de uma forma que persista ao longo do tempo. Para isso você pode usar <a href="https://docs.deta.sh/docs/base/py_tutorial?ref=fastapi" class="external-link" target="_blank">Deta Base</a>, também tem um generoso **nível gratuito**.
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:238]:
```suggestion
Em algum momento, você provavelmente desejará armazenar alguns dados para seu aplicativo de uma forma que persista ao longo do tempo. Para isso você pode usar <a href="https://docs.deta.sh/docs/base/py_tutorial?ref=fastapi" class="external-link" target="_blank">Deta Base</a>, que também tem um generoso **nível gratuito**.
```

File: docs/pt/docs/deployment/deta.md [hunk 13]
```diff
@@ -0,0 +1,258 @@
...
+
+Você também pode ler mais na <a href="https://docs.deta.sh?ref=fastapi" class="external-link" target="_blank">documentação da Deta</a>.
+
+## Conceitos de implantação
+
+Voltando aos conceitos que discutimos em [Deployments Concepts](./concepts.md){.internal-link target=_blank}, veja como cada um deles seria tratado com o Deta:
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:244]:
```suggestion
Voltando aos conceitos que discutimos em [Deployments Concepts](./concepts.md){.internal-link target=_blank}, veja como cada um deles seria tratado com a Deta:
```

File: docs/pt/docs/deployment/deta.md [hunk 14]
```diff
@@ -0,0 +1,258 @@
...
+
+* **HTTPS**: Realizado pela Deta, eles fornecerão um subdomínio e lidarão com HTTPS automaticamente.
+* **Running on startup**: Realizado pela Deta, como parte de seu serviço.
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:247]:
```suggestion
* **Executando na inicialização**: Realizado pela Deta, como parte de seu serviço.
```

File: docs/pt/docs/deployment/deta.md [hunk 15]
```diff
@@ -0,0 +1,258 @@
...
+* **Restarts**: Realizado pela Deta, como parte de seu serviço.
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:248]:
```suggestion
* **Reinicialização**: Realizado pela Deta, como parte de seu serviço.
```

File: docs/pt/docs/deployment/deta.md [hunk 16]
```diff
@@ -0,0 +1,258 @@
...
+* **Replication**: Realizado pela Deta, como parte de seu serviço.
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:249]:
```suggestion
* **Replicação**: Realizado pela Deta, como parte de seu serviço.
```

File: docs/pt/docs/deployment/deta.md [hunk 17]
```diff
@@ -0,0 +1,258 @@
...
+* **Memory**: Limite predefinido pela Deta, você pode contatá-los para aumentá-lo.
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:250]:
```suggestion
* **Memória**: Limite predefinido pela Deta, você pode contatá-los para aumentá-lo.
```

File: docs/pt/docs/deployment/deta.md [hunk 18]
```diff
@@ -0,0 +1,258 @@
...
+* **Previous steps before starting**: Não suportado diretamente, você pode fazê-lo funcionar com o sistema Cron ou scripts adicionais.
```
Comment (LorhanSohaky) [inline docs/pt/docs/deployment/deta.md:251]:
```suggestion
* **Etapas anteriores a inicialização**: Não suportado diretamente, você pode fazê-lo funcionar com o sistema Cron ou scripts adicionais.
```

**Review by LorhanSohaky — CHANGES_REQUESTED** [review]
I think with these suggestions the text is more natural

**Review by LorhanSohaky — APPROVED** [review]

**Review by cassiobotaro — APPROVED** [review]

**Review by izaguerreiro — APPROVED** [review]

**Review by LorhanSohaky — APPROVED** [review]

**Comment by tiangolo** [comment]
Great, thank you @lsglucas ! :bow: 

And thanks for the reviews @izaguerreiro, @cassiobotaro, @LorhanSohaky ! :cake:

**Pull request merged**